src/views/MissionPlanningView.vue

```diff
@@ -1793,31 +1917,45 @@ watch(vehicleStore.coordinates, () => {
     vehicleMarker.value.bindTooltip(vehicleMarkerTooltip)
     planningMap.value.addLayer(vehicleMarker.value)
   }
+
   vehicleMarker.value.setLatLng(vehiclePosition.value)
+
+  if (followerTarget.value !== WhoToFollow.VEHICLE && prevCoords === undefined) {
+    targetFollower.follow(WhoToFollow.VEHICLE)
+  }
 })
 
 const homeMarker = ref<L.Marker>()
+
 watch(home, () => {
   if (planningMap.value === undefined) throw new Error('Map not yet defined')
 
   const position = home.value
   if (position === undefined) return
 
-  if (homeMarker.value === undefined) {
-    homeMarker.value = L.marker(position as LatLngTuple)
-    const homeMarkerIcon = L.divIcon({ className: 'marker-icon', iconSize: [16, 16], iconAnchor: [8, 8] })
-    homeMarker.value.setIcon(homeMarkerIcon)
+  if (!homeMarker.value) {
+    homeMarker.value = L.marker(position as LatLngTuple, {
+      icon: L.divIcon({ className: 'marker-icon', iconSize: [24, 24], iconAnchor: [12, 12] }),
+      draggable: true,
+    })
```

Comment:
Same case here. The only two real changes were the `draggable: true` and the icon size, but there are 9 extra lines to be reviewed.